cli/index.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var CryptoJS = require("crypto-js");
+var FileSystem = require("fs");
+const Yargs = require('yargs');
+
+const SCRIPT_URL = 'https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.9-1/crypto-js.min.js';
+const SCRIPT_TAG = '<script src="' + SCRIPT_URL + '" integrity="sha384-lp4k1VRKPU9eBnPePjnJ9M2RF3i7PC30gXs70+elCVfgwLwx1tv5+ctxdtwxqZa7" crossorigin="anonymous"></script>';
+
+const namedArgs = Yargs
+      .usage('staticrypt <input> <password> options')
+      .demandCommand(2)
+      .default({'output': null, 'title': null, 'instructions': null, 'embed': null})
+      .alias('o', 'output')
+      .alias('t', 'title')
+      .alias('i', 'instructions')
+      .alias('e', 'embed')
+      .argv;
```

Comment:
If we want to keep the previous help text and show the default values we can use something like:

```
const namedArgs = Yargs
      .usage('Usage: staticrypt <input> <password> [options]')
      .demandCommand(2)
      .option('e', {
        alias: 'embed',
        type: 'boolean',
        describe: 'Whether or not to embed crypto-js in the page (or use an external CDN)',
        default: false
      })
      .option('o', {
        alias: 'output',
        type: 'string',
        describe: 'File name / path for generated encrypted file',
        default: null
      })
      .option('t', {
        alias: 'title',
        type: 'string',
        describe: "Title for output HTML page",
        default: 'Protected Page'
      })
      .option('i', {
        alias: 'instructions',
        type: 'string',
        describe: 'Special instructions to display to the user.',
        default: null
      })
      .argv;
```